modules/islandora_google_scholar/islandora_google_scholar.module

```diff
@@ -193,16 +193,21 @@ function islandora_google_scholar_parse_date_foryear($date) {
  *   An associate array containing the name => content of the meta tags.
  */
 function islandora_google_scholar_embed_tags($tags) {
+  $weight = 1000;
   if ($tags != FALSE) {
     foreach ($tags as $name => $content) {
       if (is_array($content)) {
         foreach ($content as $key => $nested_val) {
-          $element = array(
+	  if($name == 'citation_author') {
+            $weight ++;
```

Comment:
Does not appear to be a code violation, and appears to execute correctly, yet... Having the space before the post-increment is... weird? Dunno. If we want to leave it with a space, I'm fine... Just feeling the need to comment on it. :P